src/embedded-checkout/embed-checkout.ts

```diff
@@ -4,6 +4,27 @@ import EmbeddedCheckoutOptions from './embedded-checkout-options';
 import parseOrigin from './parse-origin';
 import ResizableIframeCreator from './resizable-iframe-creator';
 
+/**
+ * Embed the checkout form in an iframe.
+ *
+ * Once the iframe is embedded, it will automatically resize according to the
+ * size of the checkout form. It will also notify the parent window when certain
+ * events have occurred. i.e.: when the form is loaded and ready to be used.
+ *
+ * ```js
+ * embedCheckout({
+ *     url: 'https://checkout/url',
+ *     container: 'container-id',
```

Comment:
before seeing the example I thought that container was any selector. Should it be called `containerId`?

Reply:
Good point. I tried to match with the name we use for payment strategies. But I guess this is a separate thing and so it doesn't hurt to use a more appropriate name.